packages/kotti-ui/source/kotti-drawer/KtDrawer.vue

```diff
@@ -1,203 +1,198 @@
 <template>
-	<transition mode="out-in" name="drawer">
-		<div class="drawer-mask" @click.self="onOutsideDrawerClick">
+	<transition mode="out-in" name="kt-drawer">
+		<div class="kt-drawer__mask" @click.self="onOutsideDrawerClick">
 			<div :class="drawerClass" :style="drawerWidth">
-				<div class="drawer-header">
+				<div class="kt-drawer__header">
 					<slot name="drawer-header" />
 				</div>
-				<div class="drawer-body">
+				<div class="kt-drawer__body">
 					<slot slot name="drawer-body" />
 				</div>
-				<div class="drawer-footer">
+				<div class="kt-drawer__footer">
 					<slot name="drawer-footer" />
 				</div>
-				<div class="drawer-handle" @click="onDrawerHandleClick">
-					<i v-if="isExpanded" class="yoco"> chevron_right </i>
-					<i v-else class="yoco"> chevron_left </i>
+				<div class="kt-drawer__handle" @click="onDrawerHandleClick">
+					<i v-if="isExpanded" class="yoco" v-text="Yoco.Icon.CHEVRON_RIGHT" />
+					<i v-else class="yoco" v-text="Yoco.Icon.CHEVRON_LEFT" />
 				</div>
 			</div>
 		</div>
 	</transition>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent, ref } from '@vue/composition-api'
+
+import { KottiDrawer } from './types'
+
+export default defineComponent<KottiDrawer.PropsInternal>({
 	name: 'KtDrawer',
 	props: {
-		closeOutside: {
-			type: Boolean,
-			default: true,
-		},
-		wide: {
-			type: Boolean,
-			default: false,
-		},
-		defaultWidth: {
-			type: String,
-			default: null,
-		},
-		expandWidth: {
-			type: String,
-			default: null,
-		},
+		disallowCloseOutside: { type: Boolean, default: false },
```

Comment:
@HansAuger I actually thought about this again

I'd say, just make it `closeOutside` and `default` as `false`

If any case, we are breaking the API, might as well just replace the usages of the accordion where `closeOutside` wasn't mentioned, to explicitly mention it, and replace the other with `false`

@FlorianWendelborn any opinions?

Reply:
I think the default should be whatever behavior we prefer. I think if there's no special reason to disallow close-outside, it should be turned on

Maybe the wording can be changed somehow else (e.g. associated to that the user doesn't want to accidentally cancel here)